config.m4

```diff
@@ -153,6 +153,7 @@ if test "$MONGODB" != "no"; then
     php_phongo.c \
     phongo_compat.c \
     src/bson.c \
```

Comment:
I plan on doing so, but it currently still has the generic BSON functions in it.